content/pricing/_index.md

```diff
@@ -3,4 +3,7 @@ title: Pricing
 meta_desc: Pulumi's open source infrastructure as code SDK and enterprise SaaS products together provide plans for teams of all sizes.
 type: page
 layout: pricing
+menu:		
+    header:		
+        weight: 3
```

Comment:
Can we make this a `2`? That'll position it closer to Products, as it was before. (Why Pulumi kind of scooted over just as we launched the home-page changes.)

Reply:
Done. I just kept the weight that it was before, but it looks like `/docs/_index.md` already had a weight of 3, and no menu item had a weight of 2. Does that sound right to you?

Reply:
Yep, that sounds right.